lib/server.coffee

```diff
@@ -43,26 +44,28 @@ class Server
   ###
   _setupRoutes: ->
     @_names()
-    if isTruthy process.env['LOCAL_TESTING']
-      @_setupLocalTestingRoutes()
-    else
+    if isTruthy process.env['CLOUDMINE']
```

Comment:
Brian and Arthur walked me off the ledge. This should be fine. However I still would like a comment that explains this and the possible states of this environment variable.